presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -17,8 +17,10 @@ These configuration options may require tuning in specific situations:
 
 * ``task.max-worker-threads``:
   Sets the number of threads used by workers to process splits. Increasing this number
-  can improve throughput, if worker CPU utilization is low, but will cause increased
-  heap space usage.
+  can improve throughput, if worker CPU utilization is low and all the threads are in use,
+  but will cause increased heap space usage. You can check the number of threads being
```

Comment:
I'd rephrase as "The number of active threads is available via the .... JMX stat".